https://bitbucket.org/thomasballinger/bpython-for-clean-patch/commits/94c4664c1303
# HG changeset patch
# User Thomas Ballinger <thomasballinger@gmail.com>
# Date 1404088731 14400
# Branch 0.13-bugfix
# Node ID 94c4664c130303b1ae53629d6ebcff04f5d5d7ae
# Parent  106c49981f5e2db325d543d24c063f9c38013023
simplest possible fix for test that changes in py3.3

diff --git a/bpython/test/test_repl.py b/bpython/test/test_repl.py
--- a/bpython/test/test_repl.py
+++ b/bpython/test/test_repl.py
@@ -1,4 +1,5 @@
 import os
+import sys
 import unittest
 from itertools import islice
 from mock import Mock
@@ -280,8 +281,12 @@
 
         self.assertTrue(self.repl.complete())
         self.assertTrue(hasattr(self.repl.completer,'matches'))
-        self.assertEqual(self.repl.completer.matches,
-            ['UnboundLocalError(', '__doc__'])
+        if sys.version_info[0] == 3 and sys.version_info[1] >= 3:
+            self.assertEqual(self.repl.completer.matches,
+                ['ChildProcessError(', 'UnboundLocalError(', '__doc__'])
+        else:
+            self.assertEqual(self.repl.completer.matches,
+                ['UnboundLocalError(', '__doc__'])
 
     # 2. Attribute tests
     def test_simple_attribute_complete(self):

